core/server/worker/src/main/java/alluxio/worker/file/DefaultFileSystemWorker.java

```diff
@@ -174,6 +175,11 @@ public void stop() {
     if (mFilePersistenceService != null) {
       mFilePersistenceService.cancel(true);
     }
+    try {
+      getExecutorService().awaitTermination(10, TimeUnit.MINUTES);
```

Comment:
Does the `getExecutorService().shutdownNow();` line need to be reordered like `DefaultBlockWorker`?